Makefile

```diff
@@ -1,34 +1,123 @@
-OPERATOR_SDK_VERSION ?= v0.18.2
-IMAGE_REGISTRY ?= quay.io/oyahud
-OPERATOR_IMAGE ?= kubevirt-ssp-operator
-IMAGE_TAG ?= latest
-OPERATOR_BUILD_ARGS ?= "-v -mod=vendor"
+# Current Operator version
+VERSION ?= 0.0.1
+# Default bundle image tag
+BUNDLE_IMG ?= controller-bundle:$(VERSION)
+# Options for 'bundle-build'
+ifneq ($(origin CHANNELS), undefined)
+BUNDLE_CHANNELS := --channels=$(CHANNELS)
+endif
+ifneq ($(origin DEFAULT_CHANNEL), undefined)
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-operator-sdk:
-	curl -JL https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o operator-sdk
-	chmod 0755 ./operator-sdk
+# Image URL to use all building/pushing image targets
+IMG ?= quay.io/oyahud/kubevirt-ssp-operator:latest
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true"
```

Comment:
will this work with openshift?

Reply:
Probably yes. I run the functional tests on my cluster that is ocp 4.5.7.

But I'm not sure if we need this parameter, maybe it can be removed.

Reply:
I removed the parameter and generated manifests are the same.